### PR TITLE
address compiling problems with POSIX/linux CSystemLinux

### PR DESCRIPTION
**Pull request by pollend**
## Description
address some compiling issues with ISystemPOSIX.

### CODE REVIEW

File: include/nbl/system/ISystemPOSIX.h [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
-#if defined(__unix__)
+#if defined(_NBL_PLATFORM_LINUX_) || defined (_NBL_PLATFORM_ANDROID_)
 class ISystemPOSIX : public ISystem
 {
     protected:
         class CCaller final : public ISystem::ICaller
         {
             public:
-                inline CCaller(ISystemPOSIX* _system) : ICaller(_system) {}
+                CCaller(ISystemPOSIX* _system) : ICaller(_system) {}
```
Comment (devshgraphicsprogramming) [inline include/nbl/system/ISystemPOSIX.h:16]: why did you remove the inline?

Comment (pollend) [reply include/nbl/system/ISystemPOSIX.h:16]: I'm not sure why, ummm
